transformer/common_data_provider.go

```diff
@@ -114,7 +114,9 @@ func (c CommonDataProvider) getNodeName() (string, error) {
 func (cd CommonData) GetResourceId(metadata fetching.ResourceMetadata) string {
 	switch metadata.Type {
 	case fetchers.ProcessResourceType, fetchers.FSResourceType:
-		return cd.generateUUID(metadata)
+		return uuid.NewV5(uuid_namespace, cd.clusterId+cd.nodeId+metadata.ID).String()
+	case fetching.ElbType, fetching.EcrType, fetching.EksType, fetching.RolePolicyType:
```

Comment:
After having a glance at the corresponding fetchers, it seems like all of these values are used as `metadata.Subtype` which means you will never get into that case as your switch checks `metadata.Type`.

Instead, you should use:
`fetching.CloudLoadBalancer`
`fetching.CloudContainerRegistry`
`fetching.CloudContainerMgmt`
`fetching.CloudIdentity`